presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/InMemoryCachingHiveMetastore.java

```diff
@@ -584,17 +584,16 @@ private Optional<List<String>> loadPartitionNames(KeyAndContext<HiveTableName> h
     }
 
     @Override
-    public List<String> getPartitionNamesByFilter(
+    public List<PartitionNameWithVersion> getPartitionNamesByFilter(
             MetastoreContext metastoreContext,
             String databaseName,
             String tableName,
             Map<Column, Domain> partitionPredicates)
     {
         if (partitionVersioningEnabled) {
             List<PartitionNameWithVersion> partitionNamesWithVersion = getPartitionNamesWithVersionByFilter(metastoreContext, databaseName, tableName, partitionPredicates);
-            List<String> result = partitionNamesWithVersion.stream().map(PartitionNameWithVersion::getPartitionName).collect(toImmutableList());
             invalidateStalePartitions(partitionNamesWithVersion, databaseName, tableName, metastoreContext);
```

Comment:
can we remove this ? since its now being done in load call ?

Reply:
invalidateStalePartitions gets deleted in the `Streamline Partition and PartitionStatistics cache invalidation` commit

Reply:
got it.